internal/cli/api/api.go

```diff
@@ -67,10 +68,21 @@ func Builder() *cobra.Command {
 	return apiCmd
 }
 
+//go:embed api_docs_long_text.txt
```

Comment:
right now we are embedding this to atlascli binary, should we move it to tools/docs instead?

Reply:
we could when we make the other changes to the tools/docs

Reply:
https://github.com/mongodb/mongodb-atlas-cli/pull/3715 was already merge (hence your conflicts) it is just a matter of adding another function

Reply:
Will implement this